test/index.js

```diff
@@ -14,6 +14,16 @@ var before = lab.before;
 var after = lab.after;
 var expect = Code.expect;
 
+var internals = {
+    headerRegex: /max-age=\d+,\s+must-revalidate(,\s+private)?/
+};
+
+internals.getTtl = function (header) {
+    var age = header.match(/(\d+)/)[0];
```

Comment:
Maybe you don't need to be too bulletproof since this is test code, but `match()` can return `null`.